### ISSUE TITLE
`Uncaught error: Call to a member function xpath()` on admin-side login

### ISSUE THREAD

**Issue by petecooper** [textpattern/textpattern#1151]
(using current 4.7.0-dev at time of issue, vanilla install, zero plugins)

### Expected behaviour

User can log in to the admin-side without error.

### Actual behaviour

User unable to log in to the admin-side, 2x internal error and debug text thrown. Error appears to originate from https://github.com/textpattern/textpattern/blob/dev/textpattern/vendors/Textpattern/Module/Help/HelpAdmin.php#L126

### Steps to reproduce

* visit `/textpattern/` or `/textpattern/index.php`
* note 2x internal error notifications and following debug text:

```

Fatal error: Uncaught Error: Call to a member function xpath() on boolean in /var/www/vhosts/textpattern.co/dev-demo/live/textpattern/vendors/Textpattern/Module/Help/HelpAdmin.php:126 Stack trace: #0 /var/www/vhosts/textpattern.co/dev-demo/live/textpattern/lib/txplib_html.php(1214): Textpattern\Module\Help\HelpAdmin::pophelp('remember_login') #1 /var/www/vhosts/textpattern.co/dev-demo/live/textpattern/include/txp_auth.php(155): popHelp('remember_login') #2 /var/www/vhosts/textpattern.co/dev-demo/live/textpattern/include/txp_auth.php(60): doLoginForm('') #3 /var/www/vhosts/textpattern.co/dev-demo/live/textpattern/index.php(184): doAuth() #4 {main} thrown in /var/www/vhosts/textpattern.co/dev-demo/live/textpattern/vendors/Textpattern/Module/Help/HelpAdmin.php on line 126
```

Screencast attached (no commentary, safe for work). Terminal window is the demo site rebuild script: nuke and repave, no plugins.

[1151.mp4.zip](https://github.com/textpattern/textpattern/files/1725726/1151.mp4.zip)

**Comment by Bloke** [comment]
Should be fixed in 34c59d5b. Please test, and thanks for the report.

**Comment by petecooper** [comment]
Fantastic -thank you @Bloke!
